_posts/guides/2020-09-05-guide_custom-hud-menus.md

```diff
@@ -32,59 +32,64 @@ To open a hud menu, use [`{{ page.ccom_show }}`](/command/{{ page.ccom_show }}).
 The parameter for this command will autofill with all the available hud menus.
```

Comment:
Line 29 is also no longer accurate (bless github for not allowing me to comment on unchanged lines)